cli/src/test/java/com/devonfw/tools/ide/tool/androidstudio/AndroidStudioTest.java

```diff
@@ -80,7 +80,6 @@ public void testAndroidStudioRun(String os, WireMockRuntimeInfo wmRuntimeInfo) t
   private void checkInstallation(IdeTestContext context) {
     // commandlet - android-studio
     assertThat(context.getSoftwarePath().resolve("android-studio/.ide.software.version")).exists().hasContent("2024.1.1.1");
-    assertThat(context.getVariables().get("STUDIO_PROPERTIES")).isEqualTo(context.getWorkspacePath().resolve("studio.properties").toString());
```

Comment:
Why was this removed?

Reply:
Setting this variable is not part of the installation process anymore

Reply:
In order to test this, we would need to mock the `ProcessContext`, run AndroidStudio and then verify that this variable was set in the mocked `ProcessContext`.

Reply:
Feel free to create a new issue to improve our tests for this. Not needed to be implemented within this PR.

Reply:
Created #607